### ISSUE TITLE
"Merge" all issues tracker?!?

### ISSUE THREAD

**Issue by jedie** [pypyjs/pypyjs#163]
Every repro has it own issues tracker. That's sometimes confusing... What's about to use only one tracker?

Move existing issues to https://github.com/pypyjs/pypyjs/issues/ is not possible, isn't it?

So what's about to create one "IMPORANT" issues on every other repro, with the instruction to use https://github.com/pypyjs/pypyjs/issues/new ?

If all existing old issues are closed -> close the issues tracker on the other repros?

**Comment by rfk** [comment]
@jedie you're right, the current situation is pretty confusing.  I should take the action to go through the other ones, copy across any outstanding issues, then disable github issues on the other repos.
